noir-projects/noir-contracts/contracts/test/counter_contract/src/main.nr

```diff
@@ -89,6 +95,35 @@ pub contract Counter {
         balance_utils::get_balance(counters.at(owner).set)
     }
 
+    #[private]
```

Comment:
Feels weird this contract is getting contaminated by all of this. Maybe time for a bespoke `TXETest` contract?